src/main/java/org/folio/marccat/enumaration/CodeListsType.java

```diff
@@ -781,5 +781,120 @@ public String toString() {
     public String toString() {
       return "VM_MATERIAL_CODE";
     }
+  },
+  SUBJECT_DESCRIPTOR {
+    @Override
```

Comment:
This is strange to me... Why do you need to override `toString()` like this if you're going to return the same value as the enum name?  By default `toString()` will return the name of the enum, so AFAICT this is kind of a waste.

For example:
```
  public static void main(String[] args) throws Exception {
    System.out.println(CodeListsType.WITH_TO_STRING);
    System.out.println(CodeListsType.WITH_TO_STRING.name());
    System.out.println(CodeListsType.WITH_TO_STRING.toString());
    
    System.out.println(CodeListsType.WITHOUT_TO_STRING);
    System.out.println(CodeListsType.WITHOUT_TO_STRING.name());
    System.out.println(CodeListsType.WITHOUT_TO_STRING.toString());
  }
```
Output:
```
WITH_TO_STRING
WITH_TO_STRING
WITH_TO_STRING
WITHOUT_TO_STRING
WITHOUT_TO_STRING
WITHOUT_TO_STRING
```
Relevant impl. snippet: 
```
public enum CodeListsType {
  WITHOUT_TO_STRING,
  WITH_TO_STRING {
    @Override
    public String toString() {
      return "WITH_TO_STRING";
    }
  },
```

Reply:
I agree with Craig.

Reply:
I have changed them